TT/resources/sketches/ToSiteWise 3/ToSiteWise_3_may08a.ino

```diff
@@ -0,0 +1,70 @@
+#include <ArduinoIoTCloud.h>
```

Comment:
Did you intentionally put the sitewise template in the repo as example?

Reply:
No, it was my error... this is a test template used in manual test. Removed, thanks.